fsd_utils/authentication/decorators.py

```diff
@@ -13,14 +14,15 @@
 from .config import config_var_auth_host
 from .config import config_var_user_token_cookie_name
 from .config import signout_route
+from .config import SupportedApp
 from .config import user_route
 from .models import User
 
 
-def _failed_redirect():
+def _failed_redirect(return_app: SupportedApp | None):
     authenticator_host = current_app.config[config_var_auth_host]
 
-    return abort(redirect(authenticator_host + signout_route))
+    return abort(redirect(_build_logout_url(authenticator_host, return_app)))
```

Comment:
Might be better to set the authenticator host as a envvar in commonconfig and using it that way instead as authenticator_host is not a value that should change